### PR TITLE
Add base dev images for mongo, redis, nodejs

### PR DESCRIPTION
**Pull request by abtris**
Added public images for mongo, redis and nodejs

https://registry.hub.docker.com/repos/apiaryio/

### CODE REVIEW

**Comment by jsarenik** [comment]
Looks good :+1:

File: mongodb-dev/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+RUN echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | tee /etc/apt/sources.list.d/10gen.list
+
+RUN apt-get update && apt-get install -y mongodb-org=2.6.7 mongodb-org-server=2.6.7 mongodb-org-shell=2.6.7 mongodb-org-mongos=2.6.7 mongodb-org-tools=2.6.7
+
+RUN mkdir -p /data/db
+
+EXPOSE 27017
+
+ENTRYPOINT usr/bin/mongod --smallfiles
```
Comment (Almad) [inline mongodb-dev/Dockerfile]: Why is the initial slash omitted?

Comment (abtris) [inline mongodb-dev/Dockerfile]: Good point, it's works without slash. I don't have any intention here. I fix in next PR.

Comment (jsarenik) [inline mongodb-dev/Dockerfile]: Fixed in https://github.com/apiaryio/docker-base-images/commit/29616678a0e6b2bae51567aba4f60bc0606e3f9d

**Pull request merged**